setup/install_upstream_keylime/keylime.spec

```diff
@@ -4,6 +4,13 @@ Release:	1
 Summary:	Dummy package preventing keylime RPM installation
 License:	GPLv2+	
 BuildArch:  noarch
+Provides: keylime-base
```

Comment:
Shouldn't these be `Conflicts`, to make sure those packages will be removed, if installed?

Reply:
I think that both ways would work.
Those packages should be removed due to a fact that our dummy keylime package has version 99 and provides them.